signing/signing_test.go

```diff
@@ -0,0 +1,64 @@
+package signing
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestValidGetResults(t *testing.T) {
+	r, _ := http.NewRequest("GET", "/projects", nil)
+	w := httptest.NewRecorder()
+
+	vars := map[string]string{
+		"host":     "github.com",
+		"orgName":  "rohankh532",
+		"repoName": "scorecard-OIDC-test",
+	}
+
+	r = mux.SetURLVars(r, vars)
+
+	// Verify that corresponding results to this repo are found.
+	GetResults(w, r)
+	assert.Equal(t, http.StatusOK, w.Code)
+
+}
+
+func TestInvalidGetResults(t *testing.T) {
+	r, _ := http.NewRequest("GET", "/projects", nil)
+	w := httptest.NewRecorder()
+
+	varMaps := [...]map[string]string{
+		{
+			"host":     "github.com",
+			"orgName":  "sfasdasdasdasdsadsafsaf",
+			"repoName": "adasdsafasfsafasf",
+		},
+		{
+			"host":     "../github.com",
+			"orgName":  "rohankh532",
+			"repoName": "scorecard-OIDC-test",
+		},
+		{
+			"host":     "malicious/github.com",
+			"orgName":  "rohankh532",
+			"repoName": "scorecard-OIDC-test",
+		},
+		{
```

Comment:
Do we lack tests for 
1. invalid repoName with valid host and orgName?
2. invalid host but valid orgName and repoName?

Reply:
Added them